app/views/guidance/dates_and_deadlines.html.erb

```diff
@@ -34,7 +34,7 @@
         <% end %>
         <% body.with_row do |row| %>
             <% row.with_cell do %>
-              <span class="no-wrap">31 January 2025</span>
+              <span class="no-wrap">28 February 2025</span>
```

Comment:
Opted for hardcoding as the page contains a lot of dates that are hard coded

Reply:
Tweaked this in https://github.com/DFE-Digital/register-trainee-teachers/pull/4905/commits/e453a3275e48cf590beed1a9c489a49acaf4096d